api/trace/api.go

```diff
@@ -82,12 +82,12 @@ type Span interface {
 	// It then adds the newly created Link to the span.
 	Link(sc core.SpanContext, attrs ...core.KeyValue)
 
-	// SpancContext returns span context of the span. Return SpanContext is usable
-	// even after the span is finished.
+	// SpanContext returns span context of the span. Return SpanContext is usable
```

Comment:
While at it, could you fix `Return SpanContext is` into `Returned SpanContext is`, please? Thanks.